rpcserver.go

```diff
@@ -3942,6 +3942,11 @@ func (r *rpcServer) ClosedChannels(ctx context.Context,
 func (r *rpcServer) LookupHtlc(ctx context.Context,
 	in *lnrpc.LookupHtlcRequest) (*lnrpc.LookupHtlcResponse, error) {
 
+	if !r.cfg.StoreFinalHtlcResolutions {
+		return nil, errors.New("cannot lookup with flag " +
```

Comment:
The comment for this RPC method mentions the not found status code. Shouldn't we return a `status.NewError()` here to do that?